### DISCUSSION TITLE
Add multiple categories at once in selection box (entries)

### DISCUSSION THREAD

**Discussion by ghost** [craftcms/cms#8419] [Ideas]
### Description
We need checkboxes next to categories names along with "select all" option.
![category](https://cloud.githubusercontent.com/assets/18059922/24923891/41b62fa6-1ef3-11e7-9cf0-2c2d61566383.png)

### Steps to reproduce

1. Try to add a category.
2. Voila!

### Additional info

- Craft version: Craft CMS 2.6.2971

**Comment by brandonkelly** [comment]
You can select multiple entries by holding the <kbd>Command</kbd> or <kbd>Ctrl</kbd> key while clicking on entries. Or you can select a range of entries by clicking on one, then <kbd>Shift</kbd>-clicking on another.

But yeah, adding checkboxes would make this feature more obvious :)

**Comment by ghost** [comment]
@brandonkelly Strange, it didn't work for me before. Also when I clicked at the categories quickly by holding the Ctrl, they added and window closed unexpectedly.

**Comment by brandonkelly** [comment]
It’s <kbd>Command</kbd> on Mac, <kbd>Ctrl</kbd> on Windows/*nix. Which one are you on?

**Comment by ghost** [comment]
@brandonkelly Linux so Ctrl. Notice that it does work for me now (it didn't before, or I probably missed it somehow). Anyway better move your attention to this bug that I mentioned above. It occurs both on Chrome as well on Firefox.

**Comment by brandonkelly** [comment]
Posted that as a new issue (#1622). FFR try to keep each issue focussed on a single feature request or bug report.
